debianstatic/postinst.in

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+FILE=/var/renasar/on-http/.skupack/{{name}}.tar.gz
+SKUFILE=/var/renasar/on-http/.skupack/{{name}}.sku
+CODE=`curl -s -X POST --data-binary @${FILE} -o ${SKUFILE} -w '%{http_code}' localhost:8080/api/common/skus/pack`
```

Comment:
this is leveraging the idea that rackhd will be available locally at package install time, but the upstarts of those scripts are set up explicitly to *not* start the services on install. Lines like https://github.com/RackHD/on-tftp/blob/master/debian/on-tftp.upstart#L10 in each of the application packages were added to make sure that the services wouldn't auto-start unless a specific file exists (i.e. gotta touch a file first) for each application.

I think the end result here is that on a stock, straight from "deb" install is processing, the registration will fail, requiring the user to manually re-register